breathecode/certificate/views.py

```diff
@@ -25,9 +25,28 @@
 
 @api_view(["GET"])
 @permission_classes([AllowAny])
-def get_specialties(request):
+def get_academy_specialties(request):
     items = Specialty.objects.all()
-    serializer = SpecialtySerializer(items, many=True)
+
+    like = request.GET.get("like", None)
+    if like:
```

Comment:
is there any broken endpoint due to these migrations?

Reply:
I run all test and fix it with the syllabus_list field